src/main/java/com/sparta/popupstore/domain/promotionevent/service/PromotionEventService.java

```diff
@@ -49,21 +53,25 @@ public PromotionEventFindOneResponseDto findOnePromotionEvent(Long promotionEven
 
     @Transactional
     public PromotionEventUpdateResponseDto updatePromotionEvent(
-            PromotionEventUpdateRequestDto promotionEventUpdateRequestDto
-            , Long promotionEventId
+            PromotionEventUpdateRequestDto promotionEventUpdateRequestDto,
```

Comment:
```suggestion
            PromotionEventUpdateRequestDto requestDto,
```
해당 메서드 내에서는 requestdto가 하나만 있으니 이렇게 줄여서 써도 될 것 같습니다.

Reply:
넵 !